src/main.rs

```diff
@@ -225,7 +228,12 @@ fn main() {
     }
 }
 
-fn process_solve(theory: &Theory, bound: Option<BoundCommand>, strategy: StrategyOption, count: Option<i32>, color: bool) {
+fn process_solve(theory: &Theory,
```

Comment:
Huh, I wonder why rust-fmt didn't complain about this formatting...

Reply:
No idea. What is the conventional way of breaking lines like this?

Reply:
```rust
fn process_solve(
    theory: &Theory,
    bound: Option<BoundCommand>,
    // ...
) {
```